### DISCUSSION TITLE
[3.x] Tailwindcss not working (Laravel Vite)

### DISCUSSION THREAD

**Discussion by xuanxiaoyu69** [tailwindlabs/tailwindcss#9333] [Help] [answered]
What version of Tailwind CSS are you using?

v3.1.8

What build tool (or framework if it abstracts the build tool) are you using?

Laravel 9.29.0
Laravel Vite 3.1.0

What version of Node.js are you using?

14.19.0

What browser are you using?

Chrome

What operating system are you using?

Homestead in macOS

Screenshot:

![page1](https://user-images.githubusercontent.com/23549070/190305326-481dfac6-c723-40a7-bf36-8bbf7d5d872e.png)
![B443C2B2-8230-4B2A-BAB2-81D47E222127](https://user-images.githubusercontent.com/23549070/190305333-db9dbd7d-538f-4f8a-b0ca-989ba5e70a90.png)

macOs ip is 192.168.1.60

Homestead ip is 192.168.1.200

The program is running in Homestead, but tailwindcss uses 127.0.0.1. 

How can I modify it to use tailwindcss normally?

**Comment by wongjn** [comment, marked as answer]
Not familiar with Laravel or Laravel Vite but have tried [changing the host](https://laravel-vite.dev/guide/essentials/development.html#changing-the-host)?

**Reply by xuanxiaoyu69** [reply]
oh! thinks!
I thought I was going to modify the host in tailwind, but I never thought it was in vite!
